### DISCUSSION TITLE
TLSA Record needed?

### DISCUSSION THREAD

**Discussion by Tockra** [Mailu/Mailu#3367] [General]
Hello,

I've a setup with letsencrypt. 
I configured everything from the https://mx.domain.de/admin/domain/details/domain.de page. 
But the hoster where I'm hosting my domain (DNS server) does not support TLSA records.
Now I'm asking myself if I need to find a different dns provider which provides tlsa records or does my setup work fine without?

T

**Comment by nextgens** [comment]
It will work fine without. If you care about receiving emails securely you should consider setting up MTA-STS too (see https://mailu.io/2024.06/faq.html#how-do-i-setup-a-mta-sts-policy)

**Reply by Tockra** [reply]
Why is this not activated by default?

**Reply by nextgens** [reply]
It's like DANE; we document how to configure it but can't create the appropriate DNS records for you.

**Reply by Tockra** [reply]
I'm not asking for doing it for me. 
I'm asking why this record is not mentioned on the big page, which contains all necessary records:
https://mx.domain.de/admin/domain/details/domain.de

**Reply by nextgens** [reply]
Send a PR to make it happen if it bothers you.
https://mailu.io/2024.06/contributors/workflow.html
